09-features.md

```diff
@@ -32,13 +32,13 @@ The Context column decodes as follows:
 
 | Bits  | Name                              | Description                                               | Context  | Dependencies              | Link                                                                  |
 |-------|-----------------------------------|-----------------------------------------------------------|----------|---------------------------|-----------------------------------------------------------------------|
-| 0/1   | `option_data_loss_protect`        | Requires or supports extra `channel_reestablish` fields   | IN       |                           | [BOLT #2][bolt02-retransmit]                                          |
+| 0/1   | `option_data_loss_protect`        | ASSUMED                                                   |          |                           |                                                                       |
```

Comment:
So does this mean we no longer set the bit? What about an intermediate state of just everything setting it to required? Seems strange to never exercise the flip from optional to required, then just assume everyone supports it. The assumption route would potentially lead to unclean failures, eg: one side has bit set, the other assume it's set.

Reply:
Yeah, this PR is overly ambitious.  I think we go for stages:

1. Set it to even (i.e. old peers should go away), and simply assume everyone supports it.
2. Stop setting it, now everyone supports it (but allow it to be set).
3. In theory, recycle it now everyone ignores it.  (Far future)

Remember: you can simply set an even bit and *ignore whether they set it or not*.  It's their responsibility to disconnect if they have a problem, not yours!